File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -884,6 +884,97 @@ document.Close();
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Replace-misspelled-word).
 
+## Find and replace non-breaking spaces with regular spaces
+
+The following code example illustrates how to find and replace non-breaking spaces with regular spaces in a Word document.
+
+
```

Comment:
@sharonaumao - It seems you have added two empty line here. Please remove 1 line.

Reply:
@Ramaraj-Marimuthu -I have corrected it